src/Microsoft.AspNetCore.Hosting.Abstractions/IHostingEnvironment.cs

```diff
@@ -12,7 +12,8 @@ public interface IHostingEnvironment
     {
         /// <summary>
         /// Gets or sets the name of the environment. This property is automatically set by the host to the value
-        /// of the "ASPNETCORE_ENVIRONMENT" environment variable.
+        /// of the "ASPNETCORE_ENVIRONMENT" environment variable but can be overridden with the command line argument
+        /// --environment
```

Comment:
is that opt-in specified in comments anywhere? Whether its this repository or another? I would imagine the best place to put that is here since its not specified anywhere else.